docs/workflow/testing/libraries/filtering-tests.md

```diff
@@ -68,6 +68,21 @@ This attribute returns the 'failing' category, which is disabled by default.
 ```
 Use this attribute over test methods to skip failing tests only on the specific platforms and the specific target frameworks.
 
+#### SkipOnPlatformAttribute
+This attribute is intended to disable a test permanently on a platform where an API is not available or there is an intentional difference in behavior in between the tested platform and the skipped platform.
+
+This attribute can be applied either to a test class (will disable all the tests in that class) or to a test method. It allows multiple usages on the same member.
+
+```cs
+[SkipOnPlatform(TestPlatforms platforms, string reason)]
+```
+
+Use this attribute over test methods to skip tests only on the specific target platforms. The reason parameter doesn't affect the traits but we rather always use it so that when we see this attribute we know why it is being skipped on that platform.
```

Comment:
Might be worth adding a note that if the attribute is added to the assembly level it is recommended to include IgnoreForCI in the csproj as well?